### PR TITLE
330388 master runner encrypted key

### PR DESCRIPTION
**Pull request by asimeone**
leveraging encryption library in masterrunner along with some refactoring around file reading and writing

### CODE REVIEW

**Comment by asimeone** [comment]
logic added for 'volitleData.config', and 'config.config'.  Successful deploy.

File: src/InstallerService/InstallerServiceTests/HelperTests/EncryptionFileHandlerTests.cs [hunk 1]
```diff
@@ -20,9 +22,9 @@ public void Writes_encrypted_File()
             testString.Add("bye");
 
             var fileHandlerMock = new Mock<IFileHandler>();
-            fileHandlerMock.Setup(x => x.WritBytesToFile(It.Is<string>(y => y.Equals("testfileName.encrypted")), It.IsAny<byte[]>()));
-            
-            new EncryptionFileHandler(fileHandlerMock.Object).Write("testFileName", testString);
+            new EncryptionFileHandler(fileHandlerMock.Object).Write(@"C:\volitleData.config", testString);
+
+            fileHandlerMock.Verify(m => m.WritBytesToFile(It.Is<string>(y => y.Equals(@"C:\volitleData.encrypted")), It.IsAny<byte[]>()));
         }
```
Comment (jakebrown58) [inline src/InstallerService/InstallerServiceTests/HelperTests/EncryptionFileHandlerTests.cs]: does this actually write to disk?  if so, maybe a "." or not path would be better than writing directly to "C"

Comment (jakebrown58) [inline src/InstallerService/InstallerServiceTests/HelperTests/EncryptionFileHandlerTests.cs]: ...can get this on a later pull, as it shouldn't break anyone and the tests pass.

Comment (asimeone) [inline src/InstallerService/InstallerServiceTests/HelperTests/EncryptionFileHandlerTests.cs]: the file writing is mocked.

Comment (jakebrown58) [inline src/InstallerService/InstallerServiceTests/HelperTests/EncryptionFileHandlerTests.cs]: cool - i merged it.

**Comment by asimeone** [comment]
hey, did you happen to see the email i sent you?

**Comment by jakebrown58** [comment]
no.


what's up?


________________________________

**Pull request merged**